alacritty_terminal/src/term/mod.rs

```diff
@@ -933,16 +933,17 @@ impl Term {
                 self.colors[i] = self.original_colors[i];
             }
         }
-        if self.original_font_size != config.font.size && self.original_font_size == self.font_size
-        {
-            self.original_font_size = config.font.size;
-            self.font_size = config.font.size;
-        }
         self.visual_bell.update_config(config);
         self.default_cursor_style = config.cursor.style;
         self.dynamic_title = config.dynamic_title();
         self.auto_scroll = config.scrolling.auto_scroll;
         self.grid.update_history(config.scrolling.history() as usize, &self.cursor.template);
+
+        if self.original_font_size == self.font_size
+        {
```

Comment:
Alacritty's (and Rust's) codestyle is to open parenthesis on the same line. Running `cargo fmt` with `rustfmt` installed should fix this.

Reply:
Okay; my editor was running `rustfmt` automatically before I started working on this but it was also making a bunch of other trivial changes (and preferring the style shown here). I switched from `stable` to `nightly` and it seems to do what you want.